cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/hack/lib/util/golang.sh

```diff
@@ -8,7 +8,7 @@ function os::golang::verify_go_version() {
 
 	local go_version
 	go_version=($(go version))
-	if [[ "${go_version[2]}" != go1.7* ]]; then
+	if [[ "${go_version[2]}" != go1.8* ]]; then
```

Comment:
Removed changes from cmd/cluster-capacity/go/src/github.com/kubernetes-incubator/cluster-capacity/hack/lib/util/golang.sh and cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/hack/lib/util/golang.sh, only changing hack/lib/util/golang.sh.